### ISSUE TITLE
TypeError: context.report() called with a messageId, but no messages were present in the rule metadata

### ISSUE THREAD

**Issue by borovik96** [typescript-eslint/typescript-eslint#96]
**Repro**

```JSON
{
  "extends": ["plugin:@typescript-eslint/recommended"], // actually error happens when i add this line
  "plugins": ["@typescript-eslint/eslint-plugin"],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": 2018,
    "sourceType": "module",
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "settings": {
    "import/resolver": {
      "node": {
        "extensions": [".js", ".jsx", ".ts", ".tsx", ".json"]
      }
    }
  }
}
```

```TS
any file
```

**Expected Result**

**Actual Result**

**Additional Info**
```
[Error - 11:09:59 PM] ESLint stack trace:
[Error - 11:09:59 PM] TypeError: context.report() called with a messageId, but no messages were present in the rule metadata.
    at args (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/util/report-translator.js:252:23)
    at Object.report (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/linter.js:720:41)
    at report (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/rules/indent.js:712:21)
    at Program:exit.sourceCode.lines.forEach (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/rules/indent.js:1599:25)
    at Array.forEach (<anonymous>)
    at Program:exit (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/rules/indent.js:1560:38)
    at listeners.(anonymous function).forEach.listener (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/util/safe-emitter.js:45:58)
    at Array.forEach (<anonymous>)
    at Object.emit (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/util/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/Users/vladimir/projects/tatt-battle/client/node_modules/eslint/lib/util/node-event-generator.js:251:26)

```

**Versions**

| package                            | version  |
| ---------------------------------- | -------- |
| `@typescript-eslint/eslint-plugin` | `^0.2.1` |
| `@typescript-eslint/parser`        | `^0.2.1` |
| `typescript`                       | `^3.2.4` |

**Comment by JamesHenry** [comment]
I haven't been able to check if more than one form is supported, so this might not be it, but for plugins you only need this:

```
"plugins": ["@typescript-eslint"],
```

I can investigate in a little while if I don't hear back from you

**Comment by JamesHenry** [comment]
(Thanks, I've updated our issue templates so that they more consistently ask for that info too)

**Comment by JamesHenry** [comment]
I'll be releasing 1.0.0 shortly, please let me know if that doesn't fix the issue
